docs/design-document/03-connect/12-chatgpt-connector.md

```diff
@@ -0,0 +1,123 @@
+# ChatGPT
+
+## ChatGPT Source Connector
+
+### Configuration
+
+Before using ChatGPT source connector, you need to configure the server.
+- Please configure `sourceEnable` to `true` in `/resource/server-config.yml` to enable source functionality.
+- Please configure the source connector in `/resource/source-config.yml`:
+  - `connectorConfig`
+    - `connectorName`, name of the connector.
+    - (required) `path`, path of the API.
+    - (required) `port`, port of the API.
```

Comment:
I see that you have added default values to these 2 items in your latest revision. Why "_required_"?

Reply:
In fact, I don’t want users to use the default values ​​in the code, because I don’t know why there is no value in the yml. I prefer users to specify the configuration in the yml instead of using a black box. But your idea is also very good Logically speaking, since I didn't choose to throw an exception, it really shouldn't be "required"